tests/test_cli.py

```diff
@@ -15,14 +13,16 @@
 
 
 @pytest.fixture
-def runner():
+def root_logger():
+    return logging.getLogger()
+
+
+@pytest.fixture
+def runner(root_logger):
     r = CliRunner()
-    try:
-        with r.isolated_filesystem():
-            yield r
-    finally:
-        # Make sure other tests are not affected by the ones with --verbose
```

Comment:
This turns out to be not needed, as the callback is executed on every run, thus setting it for us.